v21.1/import-into.md

```diff
@@ -10,6 +10,7 @@ The `IMPORT INTO` [statement](sql-statements.html) imports CSV, Avro, or delimit
 
 - `IMPORT INTO` works for existing tables. To import data into new tables, read the following [Import into a new table from a CSV file](#import-into-a-new-table-from-a-csv-file) example.
 - `IMPORT INTO` cannot be used within a [transaction](transactions.html) or during a [rolling upgrade](upgrade-cockroach-version.html).
+- `IMPORT INTO` takes the table **offline** before importing the data. The table will be online again once the job has completed successfully.
```

Comment:
It's good that we're adding this note, but I wonder If 5 times on this page feels a bit on the nose stylistically? for me as a reader it's a bit much.  your call obv but my opinion is that if it's at the top under "Considerations" and at the bottom under "Known Limitations" we have definitely let the reader know.  if anything there is quite a bit of overlap between those two sections on this page and I'm wondering if we can consolidate.  At the end of the day it's more words to read that are saying the same things

also if it is technically a "known limitation" is there a GH issue we can link the user to so they know when/if this is lifted?  if not, and this is Just How It Works Now (tm) it's a shame we are deprecating `IMPORT TABLE` in favor of this which apparently does not have this limitation?  or am I wrong about that?  (Sorry to annoy but I'm reading this with my "skeptical user" hat on and feeling like I'm losing something)